flash_examples/image_embedder.py

```diff
@@ -11,31 +11,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import torch
 from torchvision.datasets import CIFAR10
 
 import flash
-from flash.core.data.data_source import DefaultDataKeys
-from flash.core.data.transforms import ApplyToKeys
 from flash.core.data.utils import download_data
 from flash.image import ImageClassificationData, ImageEmbedder
-from flash.image.embedding.transforms import IMAGE_EMBEDDER_TRANSFORMS
-from flash.image.embedding.vissl.transforms import multicrop_collate_fn
-
-# 1. Download the data and pre-process the data
-transform = IMAGE_EMBEDDER_TRANSFORMS.get("simclr_transform")()
-
-to_tensor_transform = ApplyToKeys(
-    DefaultDataKeys.INPUT,
-    transform,
-)
 
+# 1. Download the data and prepare the datamodule
```

Comment:
Can we add some tests ? Lot of code isn't covered.

Reply:
added